src/os/Transaction.h

```diff
@@ -291,12 +291,6 @@ class Transaction {
   Transaction(const Transaction& other) = default;
   Transaction& operator=(const Transaction& other) = default;
 
-  Transaction claim_and_reset() {
-    auto ret = Transaction();
-    std::swap(*this, ret);
-    return ret;
-  }
```

Comment:
I'd actually like to retain this -- there are situations where it would be useful.